### PR TITLE
Update Azure Security Center to Microsoft Defender for Cloud

### PR DESCRIPTION
**Pull request by Breanna-Stryker**
# Description

Replaced all instances of ASC/Security Center/Azure Security Center with Defender/Microsoft Defender for Cloud

Note:  Remember to add deployDefender=true to parameters when deploying to test.

## Issue reference

The issue this PR will close: #630

## Checklist

Please make sure you've completed the relevant tasks for this PR out of the following list:

* [X] All acceptance criteria in the backlog item are met
* [X] The documentation is updated to cover any new or changed features
* [X] Manual tests have passed
* [X] Relevant issues are linked to this PR

### CODE REVIEW

File: docs/images/20220204_missionlz_as_of_Feb2022_light.svg [hunk 1]
```diff
@@ -2468,8 +2468,8 @@
 							x="-3.23" dy="1.2em" class="st9">Initiatives</tspan></text>			</g>
 		</g>
 		<g id="group1360-746" transform="translate(422.293,-524.764)" v:mID="1360" v:groupContext="group">
-			<title>Security Center.1214</title>
-			<desc>Azure Security Center / Defender</desc>
+			<title>Microsoft Defender.1214</title>
+			<desc>Microsoft Defender for Cloud</desc>
```
Comment (glennmusa) [inline docs/images/20220204_missionlz_as_of_Feb2022_light.svg:2472]: Up to you if you want to make this change (we can instead push up a similar change in the source Visio in a future branch), but, L2524-2525 I think might update the green icon's label too:

https://github.com/Azure/missionlz/blob/62fa4318de3789eff2ab893135f7da67464781f8/docs/images/20220204_missionlz_as_of_Feb2022_light.svg?short_path=70556b0#L2524-L2525

Something like:

```svg
				<text x="-15.93" y="619.2" class="st8" v:langID="1033"><v:tabList/>Microsoft Defender<tspan
							x="0" dy="1.2em" class="st9">For Cloud</tspan></text>			</g>
```

Should generate:

image

**Review by glennmusa — APPROVED** [review]
Nice! Looks good out of the box. Activated Defender succesfully from the Portal and CLI. Have a comment you can choose to implement now, or we can update the source graphic later. @ me if I can be another set of eyes resolving SBOM conflicts.

**Comment by glennmusa** [comment]
#668 just went in, so merging latest `main` should kick off the generation of an accurate `_manifest` dir

File: docs/images/20220204_missionlz_as_of_Feb2022_light.svg (hunk 1 above)
Comment (Breanna-Stryker) [reply docs/images/20220204_missionlz_as_of_Feb2022_light.svg:2472]: Fixed!

**Review by glennmusa — APPROVED** [review]

**Pull request merged**